sceptre/cli/helpers.py

```diff
@@ -268,3 +269,69 @@ def default(self, item):
         :rtype: str
         """
         return str(item)
+
+
+FNS = [
+    'And',
+    'Base64',
+    'Cidr',
+    'Equals',
+    'FindInMap',
+    'GetAtt',
+    'GetAZs',
+    'If',
+    'ImportValue',
+    'Join',
+    'Not',
+    'Or',
+    'Select',
+    'Sub',
+    'Transform',
+]
+
+TAGS = [
+    'Condition',
+    'Ref'
+]
+
+
+def getatt_constructor(loader, node):
+    if isinstance(node.value, six.text_type):
+        return node.value.split('.', 1)
+    elif isinstance(node.value, list):
+        ret = loader.construct_sequence(node)
+        for item in ret:
+            if not isinstance(item, six.text_type):
+                raise ValueError("Fn::GetAtt does not support complex datastructures")
+        return ret
+    else:
+        raise ValueError("Fn::GetAtt only supports string or list values")
+
+
+def tag_constructor(loader, tag_suffix, node):
+    if tag_suffix not in FNS and tag_suffix not in TAGS:
+        raise ValueError("Bad tag: !{tag_suffix}".format(tag_suffix=tag_suffix))
```

Comment:
We could go a step further and show the list of allowed tags in the error message which might be more helpful?